EvidenceApi/V1/Controllers/BaseController.cs

```diff
@@ -25,5 +30,35 @@ public static void ConfigureJsonSerializer()
                 return settings;
             };
         }
+
+        public override void OnActionExecuting(ActionExecutingContext filterContext)
+        {
+            var userEmail = filterContext.HttpContext.Request.Headers["UserEmail"];
+            if (userEmail.Count == 0)
+            {
+                filterContext.Result = new BadRequestObjectResult("UserEmail request header empty");
+            }
+
+            var userEmailValue = userEmail[0];
+
+            if (userEmailValue == "resident-dummy-value")
```

Comment:
You could introduce a method called `skipAuditForResidentRequest` which performs this comparison

Reply:
👍